tests/test_status.py

```
@@ -155,9 +155,28 @@ def test_status_summary_only_stdout(state_env, capsys):
+        {
+            "cmd": "render",
+            "provider": "render",
+            "count": 1,
+            "timestamp": "2024-03-02T00:00:00Z",
+        }
+    )
+    env = CommandEnv(ui=DummyUI())
+    args = _status_args(json_lines=True)
+    run_status_cli(args, env)
+    payload = capsys.readouterr().out.strip()
+    assert payload.startswith("{") and payload.endswith("}")
```

Ruff (0.15.9)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)